packages/tools/client-debugger/client-debugger/api-report/client-debugger.api.md

```diff
@@ -50,7 +50,10 @@ export interface ConnectContainerMessage extends IDebuggerMessage<ConnectContain
 // @public
 export type ConnectContainerMessageData = HasContainerId;
 
-// @internal
+// Warning: (ae-incompatible-release-tags) The symbol "ConnectionStateChangeLogEntry" is marked as @public, but its signature references "StateChangeLogEntry" which is marked as @internal
+// Warning: (ae-incompatible-release-tags) The symbol "ConnectionStateChangeLogEntry" is marked as @public, but its signature references "ContainerStateChangeKind" which is marked as @internal
+//
+// @public
```

Comment:
@chentong7 We'll want to "follow the chain" here and apply the same fix for the new warnings. If we made `ConnectionStateChangeLogEntry` public, we need to make any other types it "depends on" public as well (the types it extends, the types of anything it or one of its methods (if any) might return, etc.). Can you open a PR with those?